.travis/tag-filter.sh

```diff
@@ -0,0 +1,34 @@
+TAG_GIT_URL=$(cat $PACKAGE/meta.yaml | grep "git_url" | awk '{print $2}')
+TAG_GIT_REV=$(cat $PACKAGE/meta.yaml | grep "git_rev" | awk '{print $2}')
+TAG_PATTERN=${TAG_PATTERN:-"v[0-9]*\.[0-9]*"}
```

Comment:
Should we allow any annotations after the actual version? For example `v0.1-some-note`.

BTW. How are these be sorted in the conda repository? Are we sure that `v2.0` and `v10.0` will get sorted correctly (i.e not alphabetically)?

Reply:
From what I see [conda-build documentation](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#package-version) recommends using [PEP 386](https://www.python.org/dev/peps/pep-0386/) but I'm not sure if its going to handle the leading `v` correctly.
Patterns used by `git describe` seem to follow `tcl glob` pattern syntax which means that `[0-9]*` are in fact 2 separate patterns `[0-9]` and `.*`. As you can see [here](https://travis-ci.com/SymbiFlow/conda-packages/jobs/285875635#L284) it already matches tags with extra information

Reply:
Sounds good to me. I think we can assume that we are only willing to parse tags starting with `v` and fall back to `v0.0` if the repo does something else.

Let @mithro take a look at that before we merge it though.

Reply:
You need to strip the leading 'v'.